### DISCUSSION TITLE
v1.1.13

### DISCUSSION THREAD

**Discussion by unocelli** [frangoteam/FUXA#823] [Releases]
New features:
- Switch control tag bind with bitmask
- Improvement of chart and graph property
- Alarm history progress indicator and date range picker #811
- Settings to show iframe and window at X/Y position
- URL parameter with link of page name in home (#798 #791)
- Modbus extension of connections type RTUBufferedPort, AsciiPort, UdpPort, TcpRTUBufferedPort, TelnetPort
- Save Project notification improvement (#763)
- allow-same-origin flag in iFrame to solve CORS problems using different ports (#770) @mauroalexandre
- BACnet client improvement (#750 #667 #652)
- Refactoring for rotate and move animations (#747)
- Upgrade Angular from 13 to 14 @Bykiev
- Min/Max of y-axis (#634 #736)

Fixed:
- Graph and Pipe control settings (#819)
- Copy & Paste of general element without settings (#797)
- Prevent process exit caused from script exceptions
- Numeric keypad input order (#783)
- Button toggle event of internal device (#778)
- Table of tag in realtime mode with server timestamp
- Go back from the last view after open iFrame (#753) @mauroalexandre
- Report of table with max value and negative
- Check System Tags (Connections Status) on load devices (#745)

<hr /><em>This discussion was created from the release <a href='https://github.com/frangoteam/FUXA/releases/tag/'>v1.1.13</a>.</em>

**Comment by PavloPGK** [comment]
Hello, what is the correct version of Node JS for FUXA v1.1.13 ?

**Reply by unocelli** [reply]
Hi, I would use the version 14.
